### PR TITLE
enhance: DES-3046 highlight other flag of app card

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Highlight "SimCenter" flag on App Card.

## Status

* [x] Ready

<sup>Snippets — used on [prod](https://www.designsafe-ci.org/admin/djangocms_snippet/snippet/18/change) and [next](https://designsafeci-next.tacc.utexas.edu/admin/djangocms_snippet/snippet/18/change/) — are temporary because they rely on old markup.<sup>

## Related

* [DES-3046](https://tacc-main.atlassian.net/browse/DES-3046)

## Changes

- **changed** how markup distinguishes Popular vs. SimCenter
- **added** styles for SimCenter

## Testing
1. Load `/use-designsafe/tools-applications/` ([prod](https://www.designsafe-ci.org/use-designsafe/tools-applications/), [next](https://designsafeci-next.tacc.utexas.edu/use-designsafe/tools-applications/)).
2. Verify app that `is_popular` has **yellow** flag.
3. Verify app that `is_simcenter` has **light blue** flag.

### Testing Results

| SimCenter | Popular |
| - | - |
| simcenter | popular |

The blue will be lighter than this test, cuz 3f77e63 changes the color.

### Color Ideas

| `light-2x` | `light-3x` |
| - | - |
| DS Card SimCenter Highlight - light-2x | DS Card SimCenter Highlight - light-3x |

I'd prefer something in between, but these are existing palette colors.

### CODE REVIEW

**Review by rstijerina — APPROVED** [review]

**Comment by wesleyboar** [comment]
This PR seems like one approval is enough, but —

> You’re not [authorized](https://docs.github.com/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches) to merge this pull request.

— so I will wait for two approvals or an admin merge.

**Pull request merged**
